src/cmd/ksh93/sh/init.c

```diff
@@ -1382,9 +1382,11 @@ static void put_mode(Namval_t* np, const char* val, int flag, Namfun_t* nfp)
 				mode = *(double*)val;
 		}
 		else
+		{
```

Comment:
This is a good example for why I feel quite strongly that parens should never be omitted unless the single statement block is placed on the same line as the controlling condition.